services/slam/server_test.go

```diff
@@ -173,6 +277,37 @@ func TestWorkingServer(t *testing.T) {
 			}
 			return f, nil
 		}
+		// test unary endpoint using GetPosition
```

Comment:
super nit, but could we keep the ordering consistent such that either all tests for the final API are above the corresponding tests for the temp API, or they're all below?

here they're above, but in the `t.Run` test cases above, they're below

(you won't have to rename `mockServer1` -> `mockServer3` that way either)